test/factory.js

```diff
@@ -128,6 +128,10 @@ factory.define('record', Record, Object.assign({}, trialAttributes, {
   source_url: factory.sequence((n) => `http://source.com/trial/${n}`),
   source_data: JSON.stringify({}),
 }), {
+  afterBuild: (record, attrs, callback) => {
+    record.hasTimestamps = false;
+    callback(null, record);
+  },
```

Comment:
Why are you changing it here? If it's a way to make sure we keep consistent timestamps across tests, you could use `sinon` to mock the timers. Take a look at https://github.com/opentrials/opentrials/blob/feature/contribute-data/test/agents/s3.js#L12-L22 for an example.